tests/unit/test_charm.py

```diff
@@ -1,6 +1,7 @@
-# Copyright 2023 Canonical Ltd.
-# See LICENSE file for licensing details.
+# copyright 2023 canonical ltd.
+# see license file for licensing details.
```

Comment:
Why those changes?

Reply:
Not sure how those changes got in there, good catch, I removed them.